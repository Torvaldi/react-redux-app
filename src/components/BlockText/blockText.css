.authText {
    display: flex;
    flex-direction: column;
    margin-bottom: 20px;
}

.authText_text {
    line-height: 180%;
    color: var(--darkGrey);
}

.authText_title {
    font-size: 1.8rem;
    margin-bottom: 20px;
    color: white;
}