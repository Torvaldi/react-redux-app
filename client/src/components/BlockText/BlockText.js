import React from 'react';
import propTypes from 'prop-types';
import './blockText.css';


const BlockText = (props) => {
  return (
    <article className="authText">
      <h1 className="authText_title">{props.text}</h1>
    </article>
  );
}

BlockText.prototype = {
  text : propTypes.string.isRequired,
}


export default BlockText;
