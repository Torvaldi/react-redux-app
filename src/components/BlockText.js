import React from 'react';
import { withStyles } from '@material-ui/styles';

import '../css/blockText.css';

const Blocktext = (props) => {
  return (
    <article className="authText">
      <h1 className="title">Lorem ipsum dolor sit amet consectetur </h1>
      <span className="text">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aspernatur reprehenderit velit explicabo </span>
    </article>
  );
}

export default Blocktext;
