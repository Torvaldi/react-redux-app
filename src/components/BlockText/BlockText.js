import React from 'react';
import propTypes from 'prop-types';
import './blockText.css';


const BlockText = (props) => {
  return (
    <article className="authText">
      <h1 className="title">Lorem ipsum dolor sit amet consectetur </h1>
      <span className="text">{props.text}</span>
    </article>
  );
}

BlockText.prototype = {
  text : propTypes.string.isRequired,
}


export default BlockText;
