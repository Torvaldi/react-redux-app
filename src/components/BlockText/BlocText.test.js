import React from 'react';
import { shallow, configure, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import BlockText from './BlockText';

configure({ adapter: new Adapter() });

describe('<BlockText />', () => {
    // snapshot
    it('should render Auth Componenent', () => {
        const wrapper = shallow(<BlockText />);
        expect(wrapper).toMatchSnapshot();
    });
    
    it('should render with props text', () => {
        let text = "I like cat";
        const BlockTextComponent = mount(<BlockText text={text} />);
        expect(BlockTextComponent.prop('text')).toEqual(text); 
    });
    
});