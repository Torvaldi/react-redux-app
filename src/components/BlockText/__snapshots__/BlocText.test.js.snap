// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BlockText /> should render Auth Componenent 1`] = `ShallowWrapper {}`;
